### PR TITLE
the-satori.net

### PR DESCRIPTION
**Pull request by gork7777**

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -3534,7 +3534,7 @@ intertop.ua##.click-bg-banner
 gametech.ru##.clickable_zone
 sportarena.com##.closebtn
 hdonlineporno.net##.cnuhd_center > .cnuhd_spots-wrap
-gubdaily.ru,gooosha.ru,comp-profi.com,limelady.ru##.code-block
+the-satori.netthe-satori.netthe-satori.netthe-satori.net,gubdaily.ru,gooosha.ru,comp-profi.com,limelady.ru##.code-block
```
Comment (zloyden) [inline RussianFilter/sections/specific.txt:3537]: `the-satori.netthe-satori.netthe-satori.netthe-satori.net`?

Comment (zloyden) [reply RussianFilter/sections/specific.txt:3537]: Also exchange the rule to the `the-satori.net#?#.code-block-label:contains(Реклама)`

Comment (gork7777) [reply RussianFilter/sections/specific.txt:3537]: Oh sorry I wrote it by mistake... I fixed I guess you can look into

Comment (zloyden) [reply RussianFilter/sections/specific.txt]: `the-satori.net##.code-block` blocks a useful content. Please exchange it to the rule above.

Comment (zloyden) [reply RussianFilter/sections/specific.txt]: @gork7777 I meant:
1. Delete the domain from `##.code-block` rule.
2. Add `the-satori.net#?#.code-block-label:contains(Реклама)` to the `css-extended.txt`.

Comment (gork7777) [reply RussianFilter/sections/specific.txt]: Okay fixed.

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -3534,7 +3534,8 @@ intertop.ua##.click-bg-banner
 gametech.ru##.clickable_zone
 sportarena.com##.closebtn
 hdonlineporno.net##.cnuhd_center > .cnuhd_spots-wrap
-the-satori.net,gubdaily.ru,gooosha.ru,comp-profi.com,limelady.ru##.code-block
+gubdaily.ru,gooosha.ru,comp-profi.com,limelady.ru##.code-block
+the-satori.net#?#.code-block-label:contains(Реклама)
```
Comment (zloyden) [inline RussianFilter/sections/specific.txt]: `the-satori.net#?#.code-block-label:contains(Реклама)` not for `specific.txt`, move it to the section `https://github.com/AdguardTeam/AdguardFilters/blob/master/RussianFilter/sections/css_extended.txt`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply RussianFilter/sections/specific.txt]: Ah, you did it. Now it's good, thanks)

**Review by zloyden — APPROVED** [review]

**Pull request merged**